cnd/src/http_api/amount.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{asset, asset::Erc20Quantity};
```

Comment:
Because we didn't have the contract address in cnd yet. Also, I don't think it makes sense at all given that it depends on the runtime configuration.

Also don't forget that you are looking at REST API code here, this is not the domain model.